tests/integration/tier0/non-destructive/rhsm-facts/main.fmf

```diff
@@ -0,0 +1,20 @@
+summary+: |
+    RHSM facts update when in analysis mode
+description+: |
+    Verify that the subscription-manager facts command is called after the
+    analysis is done (either succeeding or failling).
+
+/rhsm_facts_update_in_analysis:
+    summary+: |
+        RHSM facts update after analysis
+    description+: |
+        This test verifies that the RHSM facts are uploaded when running
+        convert2rhel with analysis option.
+    tag+:
+        - test-rhsm-facts-called-in-analysis
+    test: |
+        pytest -svv -m test_rhsm_facts_called_in_analysis
+    adjust+:
+        - enabled: false
+          when: distro == oracle-7, oracle-8
```

Comment:
```suggestion
          when: distro == oracle
```
The filtering calls the startswith() method, I believe, so no need to include both.
Again not a blocker, will fix in a different PR.